MOLXPCConnection.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
   s.name         = 'MOLXPCConnection'
-  s.version      = '1.2'
-  s.platform     = :osx, '10.8'
+  s.version      = '1.3'
+  s.platform     = :osx, '10.11'
```

Comment:
I don't like the idea of forcing a higher OS version just for this property. Instead, could you mark the new property as only available on 10.11 with:

```objc
@property(readonly, nonatomic, nullable) id synchronousRemoteObjectProxy API_AVAILABLE(macos(10.11));
```

That way anyone who wants to use it will need to do an availability check with:

```objc
if (@available(macos 10.11, *)) {
  [[s synchronousRemoteObjectProxy] doAThing];
}
```

Then add ```-Wpartial-availability``` to MOLXPCConnection's compiler flags